apps/myaccount/src/components/account-recovery/options/sms-recovery/sms-recovery.tsx

```diff
@@ -0,0 +1,435 @@
+                                                                        } } }
+                                                                initialValue={ editedMobile }
+                                                            />) }
+                                                    { enableEditMobile ? (
+                                                        <>
+                                                            <p>
+                                                                <Icon color="grey" floated="left" name="info circle" />
+                                                                { t( "myAccount:components.profile.forms." +
+                                                                "mobileChangeForm.inputs.mobile.note"
+                                                                ) }
+                                                            </p>
+                                                            <Form.Group inline={ true }>
+                                                                <PrimaryButton
+                                                                    size="small"
+                                                                    data-componentid={
+                                                                        `${componentId}--edit-section-form" +
+                                                                        "-submit-button`
+                                                                    }
+                                                                >
+                                                                    { t("common:update").toString() }
+                                                                </PrimaryButton>
+                                                                <SecondaryButton
+                                                                    className="link-button"
+                                                                    onClick={ handleCancel }
+                                                                    size="small"
+                                                                    data-componentid={ `${componentId}--edit-section" +
+                                                                    "-form-cancel-button` }
+                                                                >
+                                                                    { t("common:cancel").toString() }
+                                                                </SecondaryButton>
+                                                            </Form.Group>
+                                                        </>
+                                                    ):
+                                                        (
+                                                            <Form.Group inline={ true }>
```

Comment:
If not needed remove form group

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/6418/commits/0dd77c8c16b11b96a91e6b1331294e3bb8560a5b